data/init.js

```diff
@@ -178,8 +176,8 @@ const populateProdsSizes = (numProds, numSizes) => {
   })());
 };
 
-const initDB = () => {
-  return createTables()
+const initDB = () =>
+  createTables()
```

Comment:
personally found this harder to read because it looks like it spans multiple lines

Reply:
AirBnB told me to do it